addons/purchase/models/stock_config_settings.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class StockConfigSettings(models.TransientModel):
+    _inherit = 'stock.config.settings'
+
+    po_lead = fields.Float(related='company_id.po_lead', default=lambda self: self.env.user.company_id.po_lead)
+    default_new_po_lead = fields.Boolean(string="Security Lead Time for Purchase", default_model="stock.config.settings", help="Margin of error for vendor lead times. When the system generates Purchase Orders for reordering products,they will be scheduled that many days earlier to cope with unexpected vendor delays.")
+
+    @api.model
+    def get_default_new_po_lead(self, fields):
+        return {
+            'default_new_po_lead': self.env['ir.values'].get_default('stock.config.settings', 'default_new_po_lead')
+        }
+
+    @api.multi
+    def set_default_new_po_lead(self):
```

Comment:
No need to define **get_default*** and **set_default** methods, if we define field like ´default_foo` and set the `bar.settings.config` as default_model on the field definition.